### PR TITLE
Update installation docs to python 3.6

### PR DESCRIPTION
**Pull request by ricwo**
**Proposed changes**:
- update minimum version to python 3.6

**Status (please check what you already did)**:
- [ ] made PR ready for code review
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa_nlu#code-style) for instructions)

### CODE REVIEW

File: docs/user-guide/installation.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Installation
 Quick Installation
 ~~~~~~~~~~~~~~~~~~
 
-You can install both Rasa and Rasa X using pip (requires Python 3.5.4 or higher).
+You can install both Rasa and Rasa X using pip (requires Python 3.6 or higher).
```
Comment (ricwo) [inline docs/user-guide/installation.rst]:
```suggestion
You can install both Rasa and Rasa X using pip (requires Python 3.6.0 or higher).
```

**Comment by akelad** [comment]
@ricwo can you also remove the py3.5 badge from the readme?

**Comment by ricwo** [comment]
that'll be removed once the versions are updated on pypi, i.e. with the first tagged version that doesn't have 3.5 `setup.py`: https://pypi.org/project/rasa/

**Comment by akelad** [comment]
ah ofc.

**Pull request merged**